ERCS/erc-7683.md

```diff
@@ -97,13 +97,16 @@ struct ResolvedCrossChainOrder {
 	/// @dev The timestamp by which the order must be filled on the destination chain(s)
 	uint32 fillDeadline;
 
-	/// @dev The inputs to be taken from the user as part of order initiation
-	Input[] userInputs;
-	/// @dev The max outputs that the user will recieve. It's possible the actual amount depends on the state of the destination
+	/// @dev The max outputs that the filler will send. It's possible the actual amount depends on the state of the destination
 	///      chain (destination dutch auction, for instance), so these outputs should be considered a cap on filler liabilities.
-	Output[] maxOutputs;
-	/// @dev The outputs to be given to the filler as part of order settlement
-	Output[] minFillerOutputs;
+	Output[] maxSent;
+	/// @dev The minimum outputs that must to be given to the filler as part of order settlement. Similar to maxSent, it's possible
+	///      that special order types may not be able to guarantee the exact amount at initiate time, so this should be considered
+	///      a floor on filler receipts.
+	Output[] minRecieved;
```

Comment:
this looks like a typo; should be `minReceived`

Reply:
Fixed.